ci/jenkins/test-vmc.sh

```diff
@@ -0,0 +1,521 @@
+        # Matrix tests
+        echo "=== Currently ${cluster} has been live for more than 10h ==="
+        kubectl delete ns ${cluster}
+        echo "=== Old namespace ${cluster} has been deleted !!! ==="
+    done
+
+    kubectl get ns -l antrea-ci -o custom-columns=Name:.metadata.name,DATE:.metadata.creationTimestamp --no-headers=true | awk '{cmd="echo $(( $(date +%s) - $(date -d "$2" +%s) ))"; cmd | getline t ; print $1, t}' | awk '$1 ~ "whole-conformance" && $2 > 14400 {print $1}' | while read cluster; do
+        # Whole-conformance test
+        echo "=== Currently ${cluster} has been live for more than 4h ==="
+        kubectl delete ns ${cluster}
+        echo "=== Old namespace ${cluster} has been deleted !!! ==="
+    done
+
+    kubectl get ns -l antrea-ci -o custom-columns=Name:.metadata.name,DATE:.metadata.creationTimestamp --no-headers=true | awk '{cmd="echo $(( $(date +%s) - $(date -d "$2" +%s) ))"; cmd | getline t ; print $1, t}' | awk '$1 !~ "matrix" && $1 !~ "whole-conformance" && $2 > 9000 {print $1}' | while read cluster; do
+        # e2e, conformance, networkpolicy tests
+        echo "=== Currently ${cluster} has been live for more than 2.5h ==="
+        kubectl delete ns ${cluster}
+        echo "=== Old namespace ${cluster} has been deleted !!! ==="
+    done
+
+    echo "=== Auto cleanup finished ==="
+}
+
+function clean_tmp() {
+    echo "===== Clean up stale files & folders older than 7 days under /tmp ====="
+    CLEAN_LIST=(
+        "*codecov*"
+        "kustomize-*"
+        "*antrea*"
+        "go-build*"
+    )
+    for item in "${CLEAN_LIST[@]}"; do
+        find /tmp -name "${item}" -mtime +7 -exec rm -rf {} \; 2>&1 | grep -v "Permission denied" || true
+    done
+}
+
+# ensures that the script can be run from anywhere
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+GIT_CHECKOUT_DIR=${THIS_DIR}/../..
+pushd "$THIS_DIR" > /dev/null
+
+SCP_WITH_ANTREA_CI_KEY="scp -q -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -i ${GIT_CHECKOUT_DIR}/jenkins/key/antrea-ci-key"
+SSH_WITH_ANTREA_CI_KEY="ssh -q -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -i ${GIT_CHECKOUT_DIR}/jenkins/key/antrea-ci-key"
+SSH_WITH_UTILS_KEY="ssh -q -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -i ${WORKDIR}/utils/key"
+SCP_WITH_UTILS_KEY="scp -q -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -i ${WORKDIR}/utils/key"
+
+clean_tmp
+if [[ "$RUN_GARBAGE_COLLECTION" == true ]]; then
+    garbage_collection
+    exit 0
+fi
+
+if [[ "$RUN_SETUP_ONLY" == true ]]; then
+    setup_cluster
+    # deliver_antrea
+    exit 0
+fi
+
+if [[ "$RUN_CLEANUP_ONLY" == true ]]; then
+    cleanup_cluster
+    exit 0
+fi
+
+if [[ "$TESTCASE" != "e2e" && "$TESTCASE" != "conformance" && "$TESTCASE" != "all-features-conformance" && "$TESTCASE" != "whole-conformance" && "$TESTCASE" != "networkpolicy" && "$TESTCASE" != "integration" && "$TESTCASE" != "multicluster-integration" ]]; then
```

Comment:
Maybe no need to specify and check TESTCASE. There is only e2e case in Theia. There is no conformance, all-features-conformance, networkpolicy and multi-cluster test cases.